gui/include/scanHelper.hh

```diff
@@ -126,6 +128,15 @@ private:
      * @param sec Scan section beeing read
      */
     void createConfigs(unsigned condition, eudaq::ConfigurationSP conf, std::vector<ScanSection> sec);
```

Comment:
If I am not blind you have removed the function in the cpp. Why is it still here? :)